font-types/src/raw.rs

```diff
@@ -68,6 +68,21 @@ mod sealed {
     }
 }
 
+/// A trait for types that contain a byte slice which can be decoded to a scalar value.
+pub trait BytesWrapper: Sized {
```

Comment:
If we keep this I would prefer to call this something like `EndianWrapper` or something to more clearly communicate that this is a common interface for the `LittleEndian` and `BigEndian` types. (I'd also clarify this in the docs)

Reply:
The other thing that this type encodes is that the underlying byte slice doesn't have the same alignment requirements as the scalar value it decodes to. Not sure how important that is to mention, though.